src/Graph.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.
+ * Licensed under the MIT License.  See License in the project root
+ * for license information.
+ */
+
+
+namespace Microsoft\Graph;
+
+use Microsoft\Graph\Http\AbstractGraphClient;
+
+/**
+ * Class Graph
+ * @package Microsoft\Graph
+ * @copyright 2021 Microsoft Corporation
+ * @license https://opensource.org/licenses/MIT MIT License
+ * @link https://developer.microsoft.com/graph
+ */
+class Graph extends AbstractGraphClient
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function getSdkVersion(): string
+    {
+        return GraphConstants::SDK_VERSION;
```

Comment:
Just a side note and no action for this PR: we need to set the SDK_VERSION into the SdkVersion header like as specified in https://github.com/microsoftgraph/msgraph-sdk-design/blob/master/middleware/TelemetryHandler.md#requirements. 

Eventually we should emit something like:

SdkVersion: graph-php-core/1.16.0 (featureUsage=0f), graph-php/1.15.0

Reply:
This is currently handled in Core and tested for [v1](https://github.com/microsoftgraph/msgraph-sdk-php-core/blob/dev/tests/Http/Request/GraphRequestTest.php#L91) and [beta](https://github.com/microsoftgraph/msgraph-sdk-php-core/blob/dev/tests/Http/Request/GraphRequestTest.php#L102)